packages/app-headless-cms/src/modules/pageTemplates/views/editor/PageTemplateEditor.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { Editor } from "~/admin/components/ContentModelEditor/Editor";
+import { useRouter } from "@webiny/react-router";
+import { useCms } from "~/admin/hooks";
+import { CmsModel } from "~/types";
+import { EditorPlugins } from "./EditorPlugins";
+import { ModelFieldPlugins } from "./ModelFieldPlugins";
+
+type QueryMatch = Pick<Partial<CmsModel>, "modelId">;
+
+const PageTemplateEditor: React.FC = () => {
+    const { params } = useRouter();
+    const { apolloClient } = useCms();
+
+    const { modelId } = (params ? params : {}) as QueryMatch;
+    if (!apolloClient || !modelId) {
+        return null;
+    }
+    return (
+        <ModelFieldPlugins>
+            <EditorPlugins />
+            <Editor modelId={modelId} apolloClient={apolloClient} />
+        </ModelFieldPlugins>
+    );
+};
+
+export default PageTemplateEditor;
```

Comment:
Didn't we say we will avoid `export default`?

Reply:
This is loaded using `React.lazy`, so it expects a default export, I'll see if I can do some voodoo on the import statement to allow a named export here.